MekHQ/data/universe/awards/standard.xml

```diff
@@ -0,0 +1,552 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<awards xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<award>
+		<name>Medal of Honor</name>
+		<description>Distinguished oneself conspicuously by gallantry and intrepidity at the risk of life above and beyond the call of duty.</description>
+		<medal>MedalOfHonorM.png</medal>
+		<ribbon>1-01-1-MedalOfHonor.png</ribbon>
+		<xp>10</xp>
+		<edge>3</edge>
+		<stackable>true</stackable>
+		<position>1.01.1</position>
+		<qty>1</qty>
+		<item>Hero</item>
+		<size>Individual</size>
+		<range>Scenario</range>
+	</award>
+	<award>
+		<name>Combat Cross</name>
+		<description>12 pilot kills in a scenario.</description>
+		<medal>CombatCrossM.png</medal>
+		<ribbon>1-02-2-CombatCross.png</ribbon>
+		<xp>5</xp>
+		<stackable>true</stackable>
+		<position>1.02.1</position>
+		<qty>12</qty>
+		<item>Kills</item>
+		<size>Individual</size>
+		<range>Scenario</range>
+	</award>
+	<award>
+		<name>House Distinguished Service</name>
+		<description>432 battalion kills in a mission.</description>
+		<medal>HouseDistinguishedServiceM.png</medal>
+		<ribbon>1-03-1-HouseDistinguishedService.png</ribbon>
+		<xp>5</xp>
+		<stackable>true</stackable>
+		<position>1.03.1</position>
+		<qty>432</qty>
+		<item>Kills</item>
+		<size>Battalion</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Distinguished Service</name>
+		<description>12 pilot kills in a mission.</description>
+		<medal>DistinguishedServiceM.png</medal>
+		<ribbon>1-03-4-DistinguishedService.png</ribbon>
+		<xp>3</xp>
+		<stackable>true</stackable>
+		<position>1.03.4</position>
+		<qty>12</qty>
+		<item>Kills</item>
+		<size>Individual</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Silver Star</name>
+		<description>8 pilot kills in a scenario.</description>
+		<medal>SilverStarM.png</medal>
+		<ribbon>1-04-1-SilverStar.png</ribbon>
+		<xp>4</xp>
+		<stackable>true</stackable>
+		<position>1.04.1</position>
+		<qty>8</qty>
+		<item>Kills</item>
+		<size>Individual</size>
+		<range>Scenario</range>
+	</award>
+	<award>
+		<name>House Superior Service</name>
+		<description>144 company kills in a mission.</description>
+		<medal>HouseSuperiorServiceM.png</medal>
+		<ribbon>1-05-1-HouseSuperiorService.png</ribbon>
+		<xp>4</xp>
+		<stackable>true</stackable>
+		<position>1.05.1</position>
+		<qty>144</qty>
+		<item>Kills</item>
+		<size>Company</size>
+		<range>Mission</range>
+	</award>
```

Comment:
Why was this completely changed from the original's 216 battalion kills in a mission?